ssntp/server.go

```diff
@@ -181,8 +181,23 @@ func handleSSNTPClient(server *Server, conn net.Conn) {
 		switch frame.Type {
 		case COMMAND:
 			if (Command)(frame.Operand) == CONFIGURE && session.destRole.IsController() {
-				/* TODO Send the CONFIGURE payload to the config package */
+				// setup new configuration in SSNTP server
 				server.configuration.setConfiguration(frame.Payload)
+
+				// save new configuration to be persistent
+				pyld, err := configuration.Payload(frame.Payload)
+				if err != nil {
+					server.log.Errorf("Unable to retrieve configuration payload: %v\n", err)
```

Comment:
perhaps your question means if make sense to send an invalid configuration package to the scheduler? (I ask because the one who sends this new conf payload is the controller (SSNTP client) to the scheduler(SSNTP server)).

The only case(s) where scheduler may receive invalid (non-applicable) configuration, is where the ciao-controller is unable to validate its correctness prior to send the request. The only case I can think of atm is that new `scheduler.storage_uri` was given, the new route to write the configuration should be validated by the scheduler itself (controller cannot tell if scheduler is able to write into new location). If a bad uri was given, the scheduler would try to apply and when failing (e.g: new directory doesn't exists), it would just log the failure and ignore the update request.